test/stage2/arm.zig

```diff
@@ -113,4 +113,40 @@ pub fn addCases(ctx: *TestContext) !void {
             "",
         );
     }
+
+    {
+        var case = ctx.exe("addition", linux_arm);
+        // Add two numbers
+        case.addCompareOutput(
```

Comment:
That is a clever way to write the test :D

Reply:
Thanks! I figured defining an `assert` function is not possible yet, as conditional branching and comparisons are not implemented yet, so this is kind of circumvents that restriction.